toqito/perms/tests/test_antisymmetric_projection.py

```diff
@@ -33,4 +29,4 @@
 def test_antisymmetric_projection(dim, p_param, partial, expected_result):
     """Test function works as expected for a valid input."""
     proj = antisymmetric_projection(dim=dim, p_param=p_param, partial=partial).todense()
-    np.testing.assert_allclose(proj, expected_result)
+    assert np.allclose(proj, expected_result)
```

Comment:
```suggestion
    assert abs(proj - expected_result).all() <= 1E-3
```

Reply:
> So far, I am leaning towards using assert abs(test_result - expected_result).all() <= 1E-3 across all tests

Are you still leaning towards this direction? Or just for `test_antisymmetric_projection`?

Reply:
Let's make these changes on a case-by-case basis. It will be a very big PR if we do this for all-in-one PR. 

Plus, before we make these changes for all tests, we should discuss #63 with @vprusso.

Reply:
Yeah, I'd (selfishly) opt for a more piecemeal approach as I think my small brain can only handle MRs of a certain size :)